app/controllers/admin/editions_controller.rb

```diff
@@ -74,6 +74,10 @@ def confirm_export
   end
 
   def show
+    if @edition.locked?
+      redirect_to show_locked_admin_edition_path(@edition)
```

Comment:
I assume this needs a return too.

Reply:
Yes - I was having trouble suffixing `&& return` to this line but seems adding a new line with `return` works.